.gitignore

```diff
@@ -0,0 +1,30 @@
+# system files
+.DS_Store
+.ipynb_checkpoints
+venv
+.venv
+
+# folder
+temp/a
+data/
+
+# MicrosoftOffice.gitignore
+*.tmp
+
+# Word temporary
+~$*.doc*
+
+# Word Auto Backup File
+Backup of *.doc*
+
+# Excel temporary
+~$*.xls*
+
+# Excel Backup File
+*.xlk
+
+# PowerPoint temporary
+~$*.ppt*
+
+# temp requirements Python env
+requeriments.txt
```

Comment:
```suggestion
requirements.txt
```

Typo :)

Reply:
Thanks, should I commit suggestion directly in GitHub?